spec/controllers/idv/in_person/ready_to_verify_controller_spec.rb

```diff
@@ -83,11 +83,16 @@
           end
 
           context 'in_person_proofing_enforce_tmx enabled, pending fraud review,
-          enrollment not passed' do
+          enrollment not processed by USPS' do
```

Comment:
[question] since the parent context is `with enrollment`, should this actually be a sibling context?

Reply:
Both the parent context `with enrollment` and this child context have a pending in_person_enrollment.  As a result, I think it makes sense to maintain the parent-child context structure.  Let me know if you think differently.